meshtastic/mesh_interface_runtime/node_data.py

```
@@ -0,0 +1,105 @@
+        "snr",
+        "hopsAway",
+        "channel",
+        "isFavorite",
+        "lastHeard",
+        "since",
+    ]
+
+
+def filter_nodes(
+    nodes: list[dict[str, Any]],
+    include_self: bool,
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)